### PR TITLE
feat: handle bad cluster

### PR DESCRIPTION
**Pull request by florian-hoenicke**

### CODE REVIEW

File: now/cloud_manager.py [hunk 1]
```diff
@@ -18,37 +18,43 @@ def create_local_cluster(kind_path):
...
 
 def is_local_cluster(**kwargs):
+    command = f'{kwargs["kubectl_path"]} get nodes -o json'
     out, error = cmd(f'{kwargs["kubectl_path"]} get nodes -o json')
-    out = json.loads(out)
+    try:
+        out = json.loads(out)
+    except:
+        print(f'Command {command} gives the following error: {out} {error}')
+        exit(1)
```
Comment (makram93) [inline now/cloud_manager.py:57]: Why do we need to put it in try block?

Comment (florian-hoenicke) [reply now/cloud_manager.py:57]: in case, the command does not work, an error is thrown and the output is not a json.

Comment (makram93) [reply now/cloud_manager.py:57]: Then we can just print the error component in string format: 
```suggestion
    try:
        out = json.loads(out)
    except:
        print(f'Command {command} gives the following error: {error.decode("utf-8")}')
        exit(1)
```

**Comment by florian-hoenicke** [comment]
@makram93 Thanks, I accepted the suggestion you gave me

**Review by makram93 — APPROVED** [review]

**Pull request merged**